seqr/views/apis/report_api_tests.py

```diff
@@ -488,15 +539,28 @@ def test_sample_metadata_export(self, mock_google_authenticated):
         self.assertEqual(response.status_code, 403)
         self.assertEqual(response.json()['error'], 'Permission Denied')
 
+    @mock.patch('seqr.views.utils.airtable_utils.is_google_authenticated')
     @mock.patch('seqr.views.apis.report_api.datetime')
     @mock.patch('seqr.views.utils.export_utils.zipfile.ZipFile')
     @responses.activate
-    def test_gregor_export(self, mock_zip, mock_datetime):
+    def test_gregor_export(self, mock_zip, mock_datetime, mock_google_authenticated):
```

Comment:
Don't you think the `test_gregor_export` also needs to mock and test `MAX_OR_FILTERS` like `test_sample_metadata_export`?

Reply:
no you don't need to tests every possible path of  base utility functions every single time they are called. We have full coverage of that utility function, I think its actually better to test the case where there are fewer than the max samples